### PR TITLE
revert skip lm unit test

### PR DESCRIPTION
**Pull request by amir-saadat**
I was not able to reproduce the unit test issue locally so trying to check remote

### CODE REVIEW

File: jaxopt/_src/levenberg_marquardt.py [hunk 1]
```diff
@@ -330,12 +330,20 @@ def update(self, params, state: NamedTuple, *args, **kwargs) -> base.OptStep:
     # where J is jacobian and r" is second order directional derivative.
     if self.materialize_jac:
       damping_term = state.damping_factor * jnp.identity(params.size)
-      reg_hess = jnp.linalg.inv(state.jtj + damping_term)
-      velocity = reg_hess @ state.gradient
+      # Note that instead of taking the inverse of jtj_corr and multiply that
+      # by state.gradient, I am using solve which uses LU decomposition of
```
Comment (mblondel) [inline jaxopt/_src/levenberg_marquardt.py:334]:
```suggestion
      # by state.gradient, we are using `solve`, which uses LU decomposition of
```

File: jaxopt/_src/levenberg_marquardt.py [hunk 2]
```diff
@@ -330,12 +330,20 @@ def update(self, params, state: NamedTuple, *args, **kwargs) -> base.OptStep:
...
+      # jtj_corr and use that to obtain velocity. This have the advantage of
```
Comment (mblondel) [inline jaxopt/_src/levenberg_marquardt.py:335]:
```suggestion
      # jtj_corr and uses that to obtain velocity. This has the advantage of
```

File: tests/levenberg_marquardt_test.py [hunk 3]
```diff
@@ -177,6 +176,8 @@ def test_double_exponential_fit_x32(self, materialize_jac, geodesic):
         self.double_exponential_x,
         self.double_exponential_y)
 
+    print(optimize_info.params)
```
Comment (mblondel) [inline tests/levenberg_marquardt_test.py:179]: Remove

**Review by mblondel — COMMENTED** [review]
Some minor comments.

**Comment by amir-saadat** [comment]
Thanks.

**Review by mblondel — APPROVED** [review]

**Pull request merged**